Minor JWT quickstart doc updates (#45530)
quarkusio/quarkus: docs/src/main/asciidoc/security-jwt.adoc

You can use the generated key pair instead of those used in this quickstart.
====

> To generate a JSON Web Token (JWT) for the `TokenSecuredResource` endpoint, ensure that the application is running.
> 
> To do this, make sure to <<run-application>> first.

Reviewer: Sorry - this became clearer to me today:
```suggestion
Ensure the <<run-application, application is running>> before generating the JSON Web Token (JWT) for the `TokenSecuredResource` endpoint.
```
Or if it seems probable that the application isn't running.
```suggestion
<<run-application>> before generating the JSON Web Token (JWT) for the `TokenSecuredResource` endpoint.
```
